src/assets/bootstrap-grid/_functions.scss

```diff
@@ -0,0 +1,91 @@
+// Bootstrap functions
```

Comment:
Is it necessary to store as assets all these bootstrap-grid files? Are not the available at some npm module?